### DISCUSSION TITLE
[Peacock][Windows10]

### DISCUSSION THREAD

**Discussion by coskrrb2002** [idaholab/moose#18303] [Q&A Tools] [answered]
Dear moose users,

Is there any method to use peacock in windows 10?

I used ubuntu as an app in windows 10 and used Paraview to analyze output files.

But, I found a bug in Paraview to analyze my data, so I want to use Peacock to analyze my data.

If it's not possible, then I will use a virtual machine or dual booting of windows and Linux...

Best regards,
Nakkyu

**Comment by GiudGiud** [comment, marked as answer]
hello
As mentioned here, you will see some artefacts when using peacock on WSL with windows 10. 
https://mooseframework.inl.gov/getting_started/installation/windows10.html

But it should launch, using the X-server for visualization 

Guillaume

**Reply by dongcongcong606** [reply]
I ‘ve met the same question too.
